game.js

```diff
@@ -84,16 +200,16 @@ document.querySelector(".return-one-step").addEventListener("click", () => {
   const oState = game.oState;
   const xState = game.xState;
   const lastXCell = document.querySelector(
-    `[data-value = '${game.xState[xLength - 1]}']`
+    `[data-value = '${xState[xLength - 1]}']`
   );
   const lastOCell = document.querySelector(
-    `[data-value = '${game.oState[oLength - 1]}']`
+    `[data-value = '${oState[oLength - 1]}']`
   );
   if (xLength > 0) {
     if (game.xTurn) {
-      oneStepBack(lastOCell, "o", oState, (game.xTurn = false));
+      oneStepBack(lastOCell, "o", oState, game.xTurn = false);
```

Comment:
what is this game.xTurn = false ? it doesn't make sense to assign value when passing parameters to function...
did you mean game.xTurn === false ?